### PR TITLE
feat: Add custom 404 page input

### PR DESCRIPTION
**Pull request by evaristocuesta**
## Problem
Add the ability to display a customized page for 404 errors.

## Solution
Add an custom404Page input parameter in action.yml, a config variable in main.js and process this page if the requested file doesn't exist

### Checklist
- [x] The title starts either with `feat:`, `fix:`, or `chore:`
  - `feat` should be used if this pull request implements a new feature
  - `fix` should be used if this pull request fixes a bug
  - `chore` should be used for maintenance changes

### CODE REVIEW

File: http-server-action
Comment (Eun) [inline http-server-action:1]: Please remove this submodule. Was maybe just added by accident.

Comment (evaristocuesta) [reply http-server-action:1]: Yes, it was added by accident. Done!

**Comment by evaristocuesta** [comment]
Hi @Eun 

I hope you're doing well! I just wanted to kindly check in on this pull request I submitted a while ago. Whenever you get a chance, could you let me know if you plan to review it soon or if there's anything you'd like me to adjust?

I completely understand if you're busy, just wanted to make sure it doesn't fall off the radar.

Thanks so much!

**Comment by Eun** [comment]
Hey, thanks for the reminder, I am quite busy at the moment. But will review soon. 

Thanks again for the PR.

**Comment by evaristocuesta** [comment]
Hi @Eun! Just to remind you of this pull request ;)

**Review by Eun — APPROVED** [review]
looks good to me, just some nit on the added spacings

**Comment by evaristocuesta** [comment]
Thank you!

**Pull request merged**